docs/content/Getting-Started/Core/03-Create-models.mdx

```diff
@@ -1,25 +1,34 @@
 ---
-title: Getting Started with Data Modeling
-menuTitle: Getting Started
-permalink: /schema/getting-started
-category: Reference
-menuOrder: 1
-proofread: 06/18/2019
+title: Create models
+permalink: /getting-started/core/create-models
```

Comment:
This page has no counterpart in the "Cube Cloud" sequence and I honestly doubt it should be a part of "Cube Core" sequence as well. Looks like something that belongs to "data modeling" section of the docs, not here. I suggest (re)moving this page.